cpp/src/plasma/client.cc

```diff
@@ -140,13 +140,47 @@ struct ObjectInUseEntry {
   bool is_sealed;
 };
 
-struct ClientMmapTableEntry {
+class ClientMmapTableEntry {
+ public:
+  ClientMmapTableEntry(int fd, int64_t map_size)
+      : fd_(fd), pointer_(nullptr), length_(0) {
```

Comment:
```suggestion
      : fd_(fd) {
```

there's no point to those initializations, right?

Reply:
Oh I see @fsaintjacques suggested this. That's fine I guess, but what's the reason for this?

Reply:
Because if we remove the copy constructor, then move can swap in un-initialized values.